addon/styles/_frost-text.scss

```diff
@@ -15,7 +15,6 @@ $padding: 0 19px 0 6px;
 .frost-text {
   display: flex;
   align-items: center;
-  width: 175px;
```

Comment:
@sglanzer I'm guessing we don't want to have explicit `width` either on components, right?